Source/FullScreenPokemon.ts

```diff
@@ -4947,6 +4960,24 @@ module FullScreenPokemon {
         }
 
 
+        /**
+         * Starts the Player surfing.
+         *
+         * @param player   The Player.
+         * @param pokemon   The Pokemon using Strength.
+         * @todo Add the dialogue for when the Player starts surfing.
```

Comment:
Can you add back the `@todo` about replacing registerB calls with closeAllMenus?